packages/kbn-legacy-logging/src/get_logging_config.ts

```diff
@@ -72,6 +75,7 @@ export function getLoggingConfiguration(config: LegacyLoggingConfig, opsInterval
     },
     includes: {
       request: ['headers', 'payload'],
+      response: ['headers', 'payload'],
```

Comment:
This was the root of the issue in https://github.com/elastic/kibana/issues/80544: We hadn't actually configured `@hapi/good` to give the response payload, so it was always `undefined` and [therefore always calculated at 9 bytes](https://github.com/elastic/kibana/issues/80544#issuecomment-754768256).

Reply:
Good catch!